File: Java/gesbrasilburger/src/main/java/sn/brasilburger/entity/CommandeItem.java
package sn.brasilburger.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class CommandeItem {
    private int id;
    private Commande commande;
    private TypeCommandeItem type;
    private int produitId; 
    private int quantite;
    private Double prixUnitaire;
}

File: CSharp/brasilBurger/Models/CommandeItem.cs
namespace brasilBurger.Models
{
    public enum TypeCommandeItem
    {
        BURGER,
        MENU,
        COMPLEMENT
    }
    public class CommandeItem
    {
        public int Id { get; set; }
        public int CommandeId { get; set; }
        public Commande Commande { get; set; }
        public TypeCommandeItem Type { get; set; }
        public int ProduitId { get; set; }
        public int Quantite { get; set; }
        public decimal PrixUnitaire { get; set; }
    }

}